test/cogctl/actions/bundles.exs

```diff
@@ -62,7 +80,8 @@ defmodule Cogctl.Actions.Bundles.Test do
 
   test "bundles info returns an error when args are missing" do
     assert run("cogctl bundles info") =~ ~r"""
-    ERROR: "Missing required arguments"
+    Usage: [\s\S]*\
+    cogctl: Missing required arguments: 'bundle'
```

Comment:
Are we replacing `ERROR:` everywhere? If not, I'd prefer we remain as consistent as possible and leave it in to simplify parsing cogctl output.